### PR TITLE
Updates components to latest

### PR DESCRIPTION
**Pull request by bruugey**

### CODE REVIEW

File: src/components/global/Search/Search.tsx [hunk 1]
```diff
@@ -54,34 +59,19 @@ export function Search() {
...
       onChange={handleSearchChange}
-      className={css`
-        margin: ${spacing[400]}px ${spacing[600]}px;
-      `}
+      className={searchInputStyle}
     >
       {results.map(item => (
         <SearchResult
           key={item.name}
+          onClick={() => router.push(item.navPath ?? '/')}
```
Comment (shaneeza) [inline src/components/global/Search/Search.tsx:67]: I'm curious: Why did you change this?

Comment (bruugey) [reply src/components/global/Search/Search.tsx:67]: Link broke the styles - which is why that needs to be major version bumped.

**Review by shaneeza — APPROVED** [review]
